modules/sonarr_manager.py

```diff
@@ -293,6 +296,49 @@ async def list_user_shows(update: Update, context: ContextTypes.DEFAULT_TYPE):
     return CHOOSE_SHOW_TO_MANAGE
 
 
+async def list_tv_schedules(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    logger.debug(f'Firing callback: {inspect.stack()[0][3]}')
+    query = update.callback_query
+    try:
+        context.user_data.is_configured('sonarr_hostname')
+    except ValueError:
+        await context.bot.send_message(
+            chat_id=update.message.from_user.id,
+            text="You have not configured a server.Exiting."
+        )
+        return ConversationHandler.END
+    tag = next(filter(lambda x: x.get("label") == f'tg:{update.effective_user.id}',
+                      context.user_data.sonarr.get_tag_detail()), None)
+    update_info = []
+    for series_id in tag.get("seriesIds"):
+        show = context.user_data.sonarr.get_series(id_=series_id)
+        finale_air_date = datetime.strptime(show.get('lastAired'), '%Y-%m-%dT%H:%M:%SZ')
+        airing_threshold = datetime.utcnow() - timedelta(weeks=1)
+        if not show.get('ended') and finale_air_date > airing_threshold:
+            previous_episode = 'N/A' if not show.get('previousAiring') else datetime.strptime(
+                show.get('previousAiring'), '%Y-%m-%dT%H:%M:%SZ').strftime('%Y-%m-%d')
+            next_episode = 'TBD' if not show.get('nextAiring') else datetime.strptime(show.get('nextAiring'),
+                                                                                      '%Y-%m-%dT%H:%M:%SZ').strftime(
+                '%Y-%m-%d')
+            update_info.append({
+                'title': show.get('title'),
+                'previous_episode': previous_episode,
+                'next_episode': next_episode
+            })
+
+    update_info = sorted(update_info, key=lambda d: d['title'])
+    update_message = "\n\n".join(
+        [f"{x.get('title')}:\n  Previous Air Date: {x.get('previous_episode')}\n  Next Air Date: {x.get('next_episode')}"
+         for x in update_info])
+
+    await context.bot.send_message(
+        chat_id=update.effective_chat.id,
+        text=update_message
+    )
+
+    return START
```

Comment:
the landing menu has no quit so you're stuck there forever (until the bot restarts) . you could END or send a message and add a quit button to the main menu